pkg/awsManager/vpcVpnManager.go

```diff
@@ -166,13 +179,10 @@ func DeleteELB(client clientpkg.Client, vpcID *string) error {
 		}
 
 	}
-
 	return nil
-
 }
 
-func DeleteNatgateway(client clientpkg.Client, vpcID *string) error {
-
+func deleteNatgateway(client clientpkg.Client, vpcID *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file